### ISSUE TITLE
I think this project might be a little too big

### ISSUE THREAD

**Issue by pjohnmeyer** [hadelesko/liftoff-assignments#6]
Ask yourself, what is the core feature of your application that would make it valuable to users? Try eliminating one feature, and ask "Would this be valuable without this feature?" If so, maybe try eliminating another one.

**Comment by hadelesko** [comment]
That is true. I started  and see it that.  How do you think about another project that i started also. This is named sanitation-schedule. I will  send you its link of the read me.

**Comment by hadelesko** [comment]
https://github.com/hadelesko/schedule-and-sanitation.git
How do you think about it?

**Comment by pjohnmeyer** [comment]
I'm extremely concerned about switching projects at this point in the class. Feel free to reach me on Slack, otherwise let's discuss at class tomorrow.
